pkg/api/v1beta1/types.go

```diff
@@ -1079,7 +1079,7 @@ type BoundPods struct {
 	Host string `json:"host" description:"name of a node that these pods were bound to"`
 
 	// Items is the list of all pods bound to a given host.
-	Items []BoundPod `json:"items" description:"list of all pods bound to a given host"`
+	Items []Pod `json:"items" description:"list of all pods bound to a given host"`
```

Comment:
I'm not sure if it is okay to redefine v1beta1.
Do any apiserver apis use this type?